randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -1123,7 +1127,7 @@ Extra - room_id: [33]
       Trivial
   > Door to Entrance Hub
       Any of the following:
-          High Jump or Screw Attack or Space Jump or Movement (Advanced) or Can Freeze Enemies
+          Screw Attack or Space Jump or Combat (Intermediate) or Normal Damage ≥ 34 or Can Freeze Enemies
```

Comment:
Why remove the HJ requirement, from testing HJ seems to make this damage-less jump possible?
Same comment above as to the combat vs movement trick usage.

Reply:
I looked over vods of the 2021 rando tourney, and there are examples of both. They look equally tight. High jump's higher vertical speed has Samus hitting the ceiling early, cutting the jump short, so because there's no arc, it nearly breaks even compared to normal.

Reply:
Not what I experienced personally but will trust your judgement and analysis.